nemo/collections/asr/parts/submodules/squeezeformer_modules.py

```diff
@@ -171,13 +183,76 @@ def forward(self, x, att_mask=None, pos_emb=None, pad_mask=None):
 
         if self.is_adapter_available():
             # Call the adapters
-            x = self.forward_enabled_adapters(x)
+            pack_ip = {
+                'x': x,
+                'loc': 'post',
+            }
+            pack_ip = self.forward_enabled_adapters(pack_ip)
+            x = pack_ip['x']
 
         if self.is_access_enabled() and self.access_cfg.get('save_encoder_tensors', False):
             self.register_accessible_tensor(name='encoder', tensor=x)
 
         return x
 
+    def forward_single_enabled_adapter_(
```

Comment:
Can we have a class with such methods for adopters where conformer and squeezeformer can inherit from?

Reply:
Hmm can be done for example as AttentionAdapterModuleMixin (Inherits AdapterModuleMixin) and then Conformer and Squeezeformer inherits this class. Can be left as a todo. for another PR because this is quite large.